packages/protocol/src/data/workflow-data.ts

```diff
@@ -1,26 +1,26 @@
 import { WfActivatorType, WfFieldType, WfLevel } from './inscription';
 
-export const RESPONSIBLE_TYPE: Record<WfActivatorType, string> = {
+export const RESPONSIBLE_TYPE = {
   ROLE: 'Role',
   ROLE_FROM_ATTRIBUTE: 'Role from Attr.',
   USER_FROM_ATTRIBUTE: 'User from Attr.',
   DELETE_TASK: 'Nobody & delete'
-} as const;
+} as const satisfies Record<WfActivatorType, string>;
```

Comment:
This is something new in TS since 5.1 or 5

The problem with the Recode before was that the values were no longer known (ts forgots about it and only us string as value). But it gives you the typesafty, that no other key as on of WfActivatorType can be added to this object.

With satisfies this is also guaranteed, but without the loose of the specific values.